LICENSE

```diff
@@ -0,0 +1,668 @@
+GNU Affero General Public License version 3 – OpenPaaS
```

Comment:
Please be more precise about what you mean by compatible. As we are not providing a patch, there is not reason it is not compatible (else our james-gatling project is not compatible and their highcharts project is highly incompatible).
About the choice of the licence, I agree the AGPL seems not to be the more adequate. Maybe I can switch to Apache.